src/testing/factories/juju/jimm.ts

```diff
@@ -15,17 +15,21 @@ import {
   type CrossModelQueryOffer,
 } from "./jimm-types";
 
-export const auditEventFactory = Factory.define<AuditEvent>(() => ({
+export const auditEventBaseFactory = Factory.define<AuditEvent>(() => ({
```

Comment:
Instead of adding this second factory, I would just have one that only has the required fields (the same as this base factory but named `auditEventFactory`) and then the tests can provide any additional optional attributes.